src/openrct2/ride/Ride.cpp

```diff
@@ -4701,23 +4701,21 @@ static void vehicle_unset_update_flag_b1(rct_vehicle* head)
 static void ride_create_vehicles_find_first_block(Ride* ride, CoordsXYE* outXYElement)
 {
     rct_vehicle* vehicle = GET_VEHICLE(ride->vehicles[0]);
```

Comment:
This functions starts at the current track element that the vehicle is on and walks backwards up the track until it finds the first block brake or station or lift hill. I suggest we rename the `first` variables to `curTrack` as I feel it is more descriptive.

Reply:
Done!